src/source/file/file_server.rs

```diff
@@ -1,20 +1,17 @@
+extern crate mio;
```

Comment:
Marker to remove redundant `extern crate mio` in favor of the one in `lib.rs`.

Reply:
Replaced with a use statement.